README.md

```diff
@@ -96,8 +99,9 @@ The evaluation phase requires that the same environment variables be set as in t
 evaluation using the roslaunch command
 
 ```bash
+source /usr/share/gazebo/setup.sh
```

Comment:
We're not telling folks to source /opt/ros/dashing/setup.bash, isn't gazebo's setup.sh the equivalent of that?

Reply:
I think `source simulation_ws/install/setup.sh` covers `source /opt/ros/dashing/setup.bash`. If we were telling folks to `source simulation_ws/install/local_setup.sh`, then `source /opt/ros/dashing/setup.bash` is probably needed.